### PR TITLE
PP-8800: get solution licenses by status

### PR DESCRIPTION
**Pull request by fabcos92**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira](https://livechatinc.atlassian.net/browse/PP-8798)

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

**Review by hubert-wyszynski — APPROVED** [review]

File: src/pages/monetization/partner-program-api/index.mdx [hunk 1]
```diff
@@ -2540,6 +2540,7 @@ curl --request POST \
 
 - [Create License](#create-license)
 - [Get Solution Licenses](#get-solution-licenses)
+- [Get Solution Licenses by status](#get-solution-licenses-by-status)
```
Comment (oliwiapolec) [inline src/pages/monetization/partner-program-api/index.mdx:2543]:
```suggestion
- [Get Solution Licenses by Status](#get-solution-licenses-by-status)
```

File: src/pages/monetization/partner-program-api/index.mdx [hunk 2]
```diff
@@ -2691,6 +2692,87 @@ curl --request GET \
 
 </Section>
 
+<Section>
+
+<Text>
+
+## Get Solution Licenses by status
```
Comment (oliwiapolec) [inline src/pages/monetization/partner-program-api/index.mdx:2699]:
```suggestion
## Get Solution Licenses by Status
```

File: src/pages/monetization/partner-program-api/index.mdx (hunk 1 above)
Comment (oliwiapolec) [reply src/pages/monetization/partner-program-api/index.mdx:2543]: Just to keep it consistent with the naming of other methods 😌

File: src/pages/monetization/partner-program-api/index.mdx (hunk 2 above)
Comment (oliwiapolec) [reply src/pages/monetization/partner-program-api/index.mdx:2699]: same as above

File: src/pages/monetization/partner-program-api/index.mdx [hunk 3]
```diff
@@ -2691,6 +2692,87 @@ curl --request GET \
...
+- `date_from` - **required** - get a report for licenses created from a given date
+- `date_to` - **required** - get a report for licenses created up to a given date
+- `limit` - an amount of licenses in response (min. 1, max. 15 - default)
+- `next_cursor` - a next page token received in a previous request
+
+**Note:** Date format: `YYYY-MM-DDTHH:mm:ssZ`
+
+#### Response
+
+- `200 - OK` - success, see the example on the right
```
Comment (oliwiapolec) [inline src/pages/monetization/partner-program-api/index.mdx:2722]:
```suggestion
- `200 - OK` - success, see the example below
```

File: src/pages/monetization/partner-program-api/index.mdx [hunk 4]
```diff
@@ -2691,6 +2692,87 @@ curl --request GET \
...
+
+#### Path Parameters
+
+- `<status>` - **required** - license status, one of: `paid`, `trial`, `expired`
+
+#### Query Parameters
+
+- `date_from` - **required** - get a report for licenses created from a given date
+- `date_to` - **required** - get a report for licenses created up to a given date
+- `limit` - an amount of licenses in response (min. 1, max. 15 - default)
```
Comment (oliwiapolec) [inline src/pages/monetization/partner-program-api/index.mdx:2715]: Is 15 the default value? I'm asking as it's not entirely clear to me here :D

File: src/pages/monetization/partner-program-api/index.mdx [hunk 5]
```diff
@@ -2691,6 +2692,87 @@ curl --request GET \
...
+- `next_cursor` - a next page token received in a previous request
```
Comment (oliwiapolec) [inline src/pages/monetization/partner-program-api/index.mdx:2716]:
```suggestion
- `next_cursor` - a next page token received from the previous request's response
```

File: src/pages/monetization/partner-program-api/index.mdx (hunk 4 above)
Comment (oliwiapolec) [inline src/pages/monetization/partner-program-api/index.mdx:2715]:
```suggestion
- `limit` - the amount of licenses in response (min. 1, max. 15 - default)
```

Comment (fabcos92) [reply src/pages/monetization/partner-program-api/index.mdx]: Yes, it's the default :) I changed it, hope it's more clear - https://github.com/livechat/livechat-public-docs/pull/1410/commits/ea0961a8ea444c96c9ebf419367b3626139c67dd#diff-1fdbc72100416ca7bcf4938c953d51e3a2520b5431ba09b1205cf0b177de152fR2715

**Review by oliwiapolec — APPROVED** [review]

**Comment by oliwiapolec** [comment]
Let me know here/Slack if this is ready to go on prod ☺️

**Pull request merged**